src/Arcus.Observability.Telemetry.Serilog/KubernetesEnricher.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Arcus.Observability.Telemetry.Serilog
+{
+    /// <summary>
+    /// Enrichment on log events that automatically adds Kubernetes information from the environment.
+    /// </summary>
+    public class KubernetesEnricher : ILogEventEnricher
+    {
+        /// <summary>
+        /// Enrich the log event.
+        /// </summary>
+        /// <param name="logEvent">The log event to enrich.</param>
+        /// <param name="propertyFactory">Factory for creating new properties to add to the event.</param>
+        public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
+        {
+            EnrichEnvironmentVariable("KUBERNETES_NODE_NAME", logEvent, propertyFactory);
+            EnrichEnvironmentVariable("KUBERNETES_POD_NAME", logEvent, propertyFactory);
+            EnrichEnvironmentVariable("KUBERNETES_NAMESPACE", logEvent, propertyFactory);
+        }
+
+        private static void EnrichEnvironmentVariable(string name, LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
+        {
+            string value = Environment.GetEnvironmentVariable(name, EnvironmentVariableTarget.Process);
+            if (!String.IsNullOrWhiteSpace(value))
+            {
+                LogEventProperty property = propertyFactory.CreateProperty(name, value);
```

Comment:
Name of property should be configurable as well

Reply:
Actually I'm not sure if this remark is correct as we'll need those to base other components on such as https://github.com/arcus-azure/arcus.observability/issues/21. What do you think?
Maybe we should go to far with it from day 1.

Maybe it's best to keep them fixed for now, but would not use the environment variable name but more something like `Namespace`, `PodName`, `NodeName`

Reply:
Yes, was also something I thought about. I thought to leave it to the consumer to pass along the required pod name to the component enricher too. 
But I agree, maybe too much for an initial version.